apps/sms/views/conversation/index.html

```diff
@@ -217,7 +217,7 @@ <h1 id="messages-header-text" aria-hidden="true"></h1>
                 <button id="messages-attach-button" aria-label="Add attachment" data-l10n-id="attach-button" type="button"></button>
               </div>
               <div class="composer-message-input-container">
-                <div id="messages-input" contentEditable="true" name="message"
+                <div id="messages-input" class="support-emojis" contentEditable="true" name="message"
```

Comment:
I meant these fields: ```.subject-composer-input``` - subject editor, ```.message-content-body``` - message content, ```.message-subject``` - subject content, ```.panel-ReportView .message-subject``` - subject field in Report view.

Basically this file contains almost the same markup as we have in index.html except for the Inbox related fields.